### PR TITLE
Set headers

### PR DESCRIPTION
**Pull request by tkleinhakisa**
Added headers to the request sent to the workers

according to http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/using-features-managing-env-tiers.html
- [x] User-Agent
- [x] X-Aws-Sqsd-Msgid
- [x] X-Aws-Sqsd-Queue
- [x] X-Aws-Sqsd-First-Received-At // TODO: convert timestamp to  ISO 8601
- [x] X-Aws-Sqsd-Receive-Count
- [x] Content-Type 
- [ ] X-Aws-Sqsd-Taskname
- [ ] X-Aws-Sqsd-Attr
- [ ] X-Aws-Sqsd-Scheduled-At
- [ ] X-Aws-Sqsd-Sender-Id

As my java skills are limited i would be very thankfull if anyone could help me finish the list

Thanks you

### CODE REVIEW

File: src/sqsd.groovy [hunk 1]
```diff
@@ -104,7 +105,15 @@ def handleMessage(String httpHost, String httpPath, String contentType, Message
         def resp = new RESTClient(httpHost).post(
                 path : httpPath,
                 body : payload,
-                contentType : contentType
+                contentType : contentType,
+                headers: [
+                    "User-Agent": "aws-sqsd/1.1",
+                    "X-Aws-Sqsd-Msgid": message.getMessageId(),
+                    "X-Aws-Sqsd-Queue": "",
```
Comment (abdielou) [inline src/sqsd.groovy:112]: `config.sqsd.queue.name`

**Comment by abdielou** [comment]
Good stuff. I can see the usefulness of `X-Aws-Sqsd-Attr-*`.... but in the end it's up to the app to decide what's useful and what's not. That being said I guess we should just include them all only if they exists.
By all I mean, they whole list and any custom attribute (`X-Aws-Sqsd-Attr-*`).

File: src/sqsd.groovy [hunk 2]
```diff
@@ -104,7 +105,15 @@ def handleMessage(String httpHost, String httpPath, String contentType, Message
         def resp = new RESTClient(httpHost).post(
                 path : httpPath,
                 body : payload,
-                contentType : contentType
+                contentType : contentType,
+                headers: [
```
Comment (abdielou) [inline src/sqsd.groovy:109]: I guess these should be included only if they exists. I would build the map first by simply iterating `message.getMessageAttributes()`.

**Comment by mortizbey** [comment]
@abdielou what do you recommend here? Is it "mergeable" already?

**Comment by abdielou** [comment]
@ortimanu not really. Check my last message. We run the risk of sending empty properties. We should only send those specified. We can either dump a `headers` object from the request or iterate and add those we care about.

**Comment by jadbox** [comment]
Msgid is a must for me to at least add that.

**Comment by abdielou** [comment]
@tkleinhakisa and @jadbox : had to modify the changes suggested originally here to only send headers included in the request. Take a look at #11

**Pull request closed without merging**